core-bundle/src/Controller/ContentElement/AbstractContentElementController.php

```diff
@@ -26,7 +26,7 @@ public function __invoke(Request $request, ContentModel $model, string $section,
         $template = $this->createTemplate($model, 'ce_'.$type);
 
         $this->addHeadlineToTemplate($template, $model->headline);
-        $this->addCssAttributesToTemplate($template, 'ce_'.$type, $model->cssID, $classes);
+        $this->addCssAttributesToTemplate($template, 'ce_'.$type, $model->cssID, $classes, $request->attributes->get('cssID'));
```

Comment:
We could replace the third parameter with `$request->attributes->get('cssID', $model->cssID)` so that no extra flag is required. The attribute has to contain the css id and classes of course.

Reply:
I like this approach more

Reply:
I intentionally didn't write the serialized/array data into the field. We could still use the fallback value, but I think it would not handle it the same if `cssID` attribute was an empty string …